test/functional/feature_bip68_sequence.py

```diff
@@ -247,11 +234,12 @@ def test_sequence_lock_unconfirmed_inputs(self):
         tx2.nVersion = 2
         tx2.vin = [CTxIn(COutPoint(tx1.sha256, 0), nSequence=0)]
         tx2.vout = [CTxOut(int(tx1.vout[0].nValue - self.relayfee * COIN), SCRIPT_W0_SH_OP_TRUE)]
-        tx2_raw = self.nodes[0].signrawtransactionwithwallet(tx2.serialize().hex())["hex"]
+        self.wallet.sign_tx(tx=tx2)
+        tx2_raw = tx2.serialize().hex()
         tx2 = tx_from_hex(tx2_raw)
```

Comment:
In 2512e1a37df42e47a1d2ea63511ba72c2ca8a92f "test: Run feature_bip68_sequence.py with MiniWallet"

This round trip is unnecessary.

Reply:
Do you mean `tx_from_hex` is unnecessary?

```diff
diff --git a/test/functional/feature_bip68_sequence.py b/test/functional/feature_bip68_sequence.py
index 28f5acedb..56f4222a9 100755
--- a/test/functional/feature_bip68_sequence.py
+++ b/test/functional/feature_bip68_sequence.py
@@ -236,7 +236,6 @@ class BIP68Test(BitcoinTestFramework):
         tx2.vout = [CTxOut(int(tx1.vout[0].nValue - self.relayfee * COIN), SCRIPT_W0_SH_OP_TRUE)]
         self.wallet.sign_tx(tx=tx2)
         tx2_raw = tx2.serialize().hex()
-        tx2 = tx_from_hex(tx2_raw)
         tx2.rehash()

         self.wallet.sendrawtransaction(from_node=self.nodes[0], tx_hex=tx2_raw)
```